static/app/views/performance/browser/webVitals/pageSamplePerformanceTable.tsx

```diff
@@ -399,9 +408,13 @@ export function PageSamplePerformanceTable({transaction, search, limit = 9}: Pro
               // Reset pagination and sort when switching datasets
               router.replace({
                 ...location,
-                query: {...location.query, sort: undefined, cursor: undefined},
+                query: {
+                  ...location.query,
+                  sort: undefined,
+                  cursor: undefined,
+                  [DATESET_KEY]: newDataSet,
```

Comment:
nit: maybe we should call this something else to not be confused with the clickhouse/storage datasets